src/test/java/org/folio/circulation/rules/Text2DroolsTest.java

```diff
@@ -16,6 +20,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import io.vertx.core.MultiMap;
+import io.vertx.core.http.CaseInsensitiveHeaders;
 import io.vertx.core.json.JsonArray;
 
 public class Text2DroolsTest {
```

Comment:
There doesn't appear to be any new tests for patron notice policies, as there were with request policies.

Is that intentional?